site/gatsby-site/src/pages/login.js

```diff
@@ -159,30 +159,33 @@ const Login = (props) => {
             </div>
           </Button>
 
-          <Button
-            variant="primary"
-            onClick={clickLoginWithGoogle}
-            className={'w-full mt-7'}
-            disabled={displayGoogleSpinner || displayFacebookSpinner}
-          >
-            <div className={'flex justify-center items-center'}>
-              {displayGoogleSpinner ? (
-                <Spinner animation="border" size="sm" role="status" aria-hidden="true" />
-              ) : (
-                <FontAwesomeIcon
-                  icon={faGoogle}
-                  color={'#ffffff'}
-                  className={'pointer fa fa-lg'}
-                  title="Login with Google"
-                />
-              )}
-              <div className={'ml-4'}>
-                <Trans ns="login">Login with Google</Trans>
+          {/* Hide Google login until brand name is supported */}
```

Comment:
Please remove this, instead of the `false` thing, the branch will be still here anyway.

Reply:
done